docs/managed_package.rst

```diff
@@ -1,27 +1,156 @@
-Release a managed package
+Release a Managed Package
 =========================
+This section outlines the various steps involved in releasing a first generation (1GP) Salesforce Managed Package project.
+Salesforce.org's Release Engineering team practices `CumulusCI Flow<TODO link>` which incorporates all of these steps.
 
 Prerequisites
 -------------
+This section assumes:
 
-Deploy to a packaging org
+* CumulusCI is installed on your computer.
+* There exists Salesforce Managed Package project configured for use with CumulusCI.
+* You have connected your packaging org to CumulusCI under the name of ``packaging``.
+
+
+
+Deploy to a Packaging Org
 -------------------------
+CumulusCI easily enables you to easily deploy to your packaging org:
+
+.. warning::
+
+    The ``ci_master`` flow runs the `uninstall_packaged_incremental <TODO>` task.
+    This task deploys destructive changes to the target org.
+
+.. code-block:: console
+
+    $ cci flow run ci_master --org packaging
+
+
+.. note::
+
+    This flow is `source format agnostic <TODO>`.
+
+The ``ci_master`` flow does the following in the target org:
+
+#. Update any project dependencies
+#. Deploy any unpackaged metadata located in the ``pre/`` directory.
+#. Deploy package metadata
+#. Deploy destructive changes to remove metadata in the target org that is no longer in the local workspace.
+
+
+
+Create a Beta Version
+---------------------
+The ``release_beta`` flow exists to group the common tasks that we want
+to execute when we release a new beta version of a project.
+
+.. code-block:: console
+
+    $ cci flow run release_beta --org packaging
+
+It is typically run against the project's packaging org, and does the following: 
+
+* Uploads new beta version to the packaging org
+* Creates a new GitHub release tag for the new beta version
+* `Generates Release Notes`_
+* Syncs feature branches with the ``main/`` branch.
```

Comment:
"syncs" is one of those words that could mean a lot of different things. It would probably be good to link to the relevant section of the CumulusCI Flow doc, and/or say _why_ this is important (so that feature branches have the latest changes from the default branch integrated)